driver/src/test/unit/com/mongodb/MongoCollectionSpecification.groovy

```diff
@@ -606,7 +606,9 @@ class MongoCollectionSpecification extends Specification {
     def 'should use FindOneAndDeleteOperation correctly'() {
         given:
         def collection = new MongoCollectionImpl(namespace, Document, codecRegistry, readPreference, writeConcern, executor)
-        def expectedOperation = new FindAndDeleteOperation(namespace, new DocumentCodec()).filter(new BsonDocument('a', new BsonInt32(1)))
+        def expectedOperation = new FindAndDeleteOperation(namespace, new DocumentCodec())
+                .filter(new BsonDocument('a', new BsonInt32(1)))
+                .writeConcern(writeConcern)
```

Comment:
Given the above comments, should add a test with at least one non-default but still acknowledged write concern.